sdk/storage/azure-storage-blobs/BREAKINGCHANGES.md

```diff
@@ -0,0 +1,9 @@
+# Release History
```

Comment:
The way we tend to do changelogs elsewhere is, instead of having a separate markdown for breaking changes, we organize it with sections for "features", "breaking changes", etc. in one page.

**For example:**
https://github.com/Azure/azure-sdk-for-c/releases/tag/1.0.0-preview.5
https://github.com/Azure/azure-sdk-for-java/releases/tag/azure-core-amqp_1.6.0

![image](https://user-images.githubusercontent.com/6527137/95930627-c3ad8700-0d7b-11eb-9374-6c971c007376.png)

That way, the changelog.md is a single document which we can link from blogs/etc. and is self-contained.

Feel free to follow that approach :)